requirements.txt

```diff
@@ -23,3 +23,6 @@ pydocstyle~=1.0.0
 cmakelint~=1.3.4
 requests_mock~=0.7.0
 vim-vint~=0.3.6
+# Stable version 3.2 is problematic in Windows.
+# Use >=3.3 once nltk-3.3 is released. 
+nltk==3.1
```

Comment:
3.1 currently worked perfectly on Windows. Any changes would require a test
again. I'll update and see how Appveyor responds..
On 10 Apr 2016 09:48, "AbdealiJK" <notifications@github.com> wrote:

> In requirements.txt
> <https://github.com/coala-analyzer/coala-bears/pull/315#discussion_r59126552>
> :
>
> > @@ -23,3 +23,6 @@ pydocstyle~=1.0.0
> >  cmakelint~=1.3.4
> >  requests_mock~=0.7.0
> >  vim-vint~=0.3.6
> > +# Stable version 3.2 is problematic in Windows.
> > +# Use >=3.3 once nltk-3.3 is released.
> > +nltk==3.1
>
> Can we use ~=3.1.0 ?
> They may have patched versions of 3.1 which go like 3.1.x later ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/315/files/9c4f632ec35a339128064e5fa9b79977ec6eda71#r59126552>
>